src/pages/Cores/LearnMore/index.tsx

```diff
@@ -0,0 +1,80 @@
+import {articlesData} from './ArticlesData';
+import {SFC} from 'types';
+import {useToggle} from 'hooks';
+import * as S from './Styles';
+import ArticleCard from './ArticleCard';
+import CoreModal from 'modals/CoreModal';
+import EmptyText from 'components/EmptyText';
+
+const LearnMore: SFC = ({className}) => {
+  const [coreModalIsOpen, toggleCoreModal] = useToggle(false);
+  const articles = articlesData;
+
+  const renderContent = () => {
+    if (!!articles.length) return renderArticles();
+    return <EmptyText>No articles to display.</EmptyText>;
+  };
+
+  const renderArticles = () => {
+    const coreCards = articles.map((article) => {
+      if (article.type === 'list') {
+        return <ArticleCard article={article} key={article.id} element={renderDataInArticles(article.data)} />;
+      } else if (article.type === 'normal') {
+        return <ArticleCard article={article} key={article.id} />;
+      } else if (article.type === 'code') {
+        return <RenderCode data={article} />;
+      }
+    });
+    return <S.CardsContainer>{coreCards}</S.CardsContainer>;
+  };
+
+  const renderDataInArticles = (data?: object) => {
+    console.log(data);
+    const dataArr = data ? Object.values(data) : [];
+    return (
+      <S.ListContainer>
+        <S.Line />
+        {dataArr.map((item, index) => (
+          <S.ListItem key={index}>
+            <S.RoleNumber>{index + 1}</S.RoleNumber>
+            <S.ListLine>
+              <S.ContentTitle>{item[0]}</S.ContentTitle>
+              <S.ContentDetail>{item[1]}</S.ContentDetail>
+            </S.ListLine>
+          </S.ListItem>
+        ))}
+      </S.ListContainer>
+    );
+  };
+
+  const RenderCode = (data: any) => {
+    const detail = data.data.detail;
+    const wordToStyle = 'Ed25519';
+
+    const parts = detail.split(new RegExp(`(${wordToStyle})`, 'gi'));
+    return (
+      <S.CodeContainer>
+        <S.ListItem>
+          <S.Img alt="tag icon" src={data.data.logo}></S.Img>
+          <S.ContentTitle>{data.data.title}</S.ContentTitle>
+        </S.ListItem>
+        <S.Code>{JSON.stringify(data.data.data, null, 2).replace(/\\n/g, '\n')}</S.Code>
+        <br />
+        <S.Detail>
+          {parts.map((part: string) =>
+            part.toLowerCase() === wordToStyle.toLowerCase() ? <S.Span>{part}</S.Span> : part.replace(/\\n/g, '\n'),
+          )}
+        </S.Detail>
+      </S.CodeContainer>
+    );
+  };
+
+  return (
+    <>
+      <S.Container className={className}>{renderContent()}</S.Container>
+      {coreModalIsOpen ? <CoreModal close={toggleCoreModal} /> : null}
```

Comment:
This code seems to be duplicated:
<img width="283" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/c5005d4a-5eae-4466-9a49-2a035d152ac1">

Can we get rid of this code duplication?

Reply:
I removed the duplicates...